schema.graphql

```diff
@@ -306,4 +306,4 @@ type WidgetCollection {
   byIdOrFirst(id: Int): Widget!
   enabled: [Widget!]!
   next(currentWidgetId: Int): Widget!
-}
```

Comment:
This will be problematic since our `graphql:schema:dump` CI step will likely fail on this

Reply:
dropped gql files from the glob pattern